slsDetectorSoftware/src/DetectorImpl.cpp

```diff
@@ -1242,25 +1242,45 @@ int DetectorImpl::acquire() {
     return OK;
 }
 
-void DetectorImpl::startAcquisition(bool blocking, std::vector<int> positions) {
-    // handle Mythen3 synchronization
-    if (shm()->detType == defs::MYTHEN3 && size() > 1) {
-        std::vector<int> master;
-        std::vector<int> slaves;
-        if (positions.empty() ||
-            (positions.size() == 1 && positions[0] == -1)) {
-            positions.resize(modules.size());
-            std::iota(begin(positions), end(positions), 0);
+void DetectorImpl::getMasterSlaveList(std::vector<int> positions,
```

Comment:
Maybe better to use const& like in the other places? 
```
//Easiest done with the using declaration we have in sls_detector_defs
using Positions = const std::vector<int> &;
```

Reply:
This is the only place where we modify positions, but will do it in the other places where we dont.